packages/config/config/devices/templates/master_template.json

```diff
@@ -71,6 +71,13 @@
 		"defaultValue": 0,
 		"allowManualEntry": false
 	},
+	"base_0-2_options_nounit": {
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 2,
+		"defaultValue": 0,
+		"allowManualEntry": false
+	},
```

Comment:
I don't think this (and the template) above make much sense to be standalone anymore, since min/max value are now inferred from the options. I'll combine them after merging this.